esbuild.mjs

```diff
@@ -10,11 +10,11 @@ const baseConfig = {
     'process.env.NODE_ENV': '"production"',
     global: 'window',
   },
-  inject: ['./esbuild-shims.js'],
-  minify: true,
+  inject: ['./esbuild-shims.mjs'],
+  minify: false,
```

Comment:
don't we want to minify?

Reply:
Good catch. I had disabled it for testing.
```suggestion
  minify: true,
```